src/Math-Tests-Matrix/PMMatrixTest.class.st

```diff
@@ -632,6 +632,56 @@ PMMatrixTest >> testPrintOn [
 	m printOn: stream
 ]
 
+{ #category : #tests }
```

Comment:
These are great tests! Well done. My only suggestions are formatting to Arrange Act Assert.